src/ui/public/vis/vis_types/angular_vis_type.js

```diff
@@ -49,6 +49,7 @@ export function AngularVisTypeProvider(Private, $compile, $rootScope) {
           updateScope();
           this.$scope.uiState = this.vis.getUiState();
           this.el.html($compile(this.vis.type.visConfig.template)(this.$scope));
+          this.$scope.$apply();
```

Comment:
Is there a specific error that occurred without that?

Reply:
removing